src/compiler/abstract_compiler.nit

```diff
@@ -188,6 +188,8 @@ class MakefileToolchain
 		var time1 = get_time
 		self.toolcontext.info("*** END WRITING C: {time1-time0} ***", 2)
 
+		if not toolcontext.check_errors then return
```

Comment:
why do we need that here by the way?

Reply:
To stop if errors on missing pkg-config packages were raised by `write_makefile`, before the C compilation.

Reply:
`write_makefile` may not be the best place for the pkg-config check. In fact, the whole check logic could be moved somewhere else, but it needs to know the mainmodule and the target platform, and having the check after the generation of the C code is useful but not vital.